src/index.js

```diff
@@ -8,7 +8,7 @@ ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root'),
+  document.getElementById('root')
```

Comment:
usually we keep this comma since it's a line-separated list, but the linter doesn't seem to catch it. either way is fine i think